src/scripts.js

```diff
@@ -0,0 +1,72 @@
+ loadJSON(function(response) {
```

Comment:
We shouldn't need to wait to load the JSON before the page kicks in, this will slow down the initial rendering. I'd say we opt for promises too.